src/mds/MDSRank.cc

```diff
@@ -2399,6 +2399,24 @@ void MDSRankDispatcher::handle_mds_map(
   if (oldmap.get_max_mds() != mdsmap->get_max_mds()) {
     purge_queue.update_op_limit(*mdsmap);
   }
+
+  {
+    auto &q = mdcache->export_pin_delayed_queue;
```

Comment:
This would be better in the MDCache code. Let's just add a notification method there like:

`void MDCache::handle_mdsmap(const MDSMap&);`

Then do this below code there.

Reply:
done